integration_tests/libccp_integration/test_datapath.c

```diff
@@ -277,11 +274,13 @@ int main(__attribute__((unused))int argc, __attribute__((unused))char **argv) {
     int recv_sock = setup_listening_thread();
     
     // initialize a fake connection; sends create messages
+	// sleep to make sure portus has started
+	sleep(2);
```

Comment:
I don't think this is necessary, but let's not worry about it now, since my PR #37 will fix it anyway.